src/renderer/angular/controllers/commons/dialogs/send-token-controller.js

```diff
@@ -267,7 +265,7 @@ function SendTokenDialogController($rootScope, $scope, $log, $q, $mdDialog, $int
         return txGenPromise;
     }
 
-    function sendEther() {
+    function send() {
```

Comment:
Please consider not to name functions in so basic way. ``sendSmartContract`` instead?